src/pydicom/dataelem.py

```diff
@@ -527,6 +527,20 @@ def _convert_value(self, val: Any) -> Any:
         if len(val) == 1:
             return self._convert(val[0])
 
+        # Some ambiguous VR elements ignore the VR for part of the value
+        # e.g. LUT Descriptor is 'US or SS' and VM 3, but the first value
+        #   is always US
+        if self.tag in _LUT_DESCRIPTOR_TAGS and val:
+
+            def _skip_conversion(val: Any) -> Any:
+                return val
+
+            validate_value(VR_.US, val[0], self.validation_mode)
+            for value in val[1:]:
```

Comment:
Technically, the third value is also always `US`, so it should also be validated. In reality, the value is always <=16, so it doesn't really matter - I guess that is the reason you did it that way?